ts/features/bonus/cgn/screens/discount/CgnDiscountDetailScreen.tsx

```diff
@@ -172,22 +154,21 @@ const CgnDiscountDetailScreen = () => {
     }
   };
 
-  const { backgroundColor } = discountDetails?.isNew
-    ? styles.backgroundNewItem
-    : styles.backgroundDefault;
+  const {
+    header: { default: defaultHeaderStyle, new: newHeaderStyle }
+  } = useCgnStyle();
+
+  const backgroundColor = discountDetails?.isNew
+    ? newHeaderStyle.backgroundColor
+    : defaultHeaderStyle.backgroundColor;
```

Comment:
As before, we can do the same thing:
```suggestion
  const { backgroundColor } = discountDetails?.isNew
    ? newHeaderStyle
    : defaultHeaderStyle;
```